worlds/ladx/__init__.py

```diff
@@ -500,8 +501,14 @@ def remove(self, state, item: Item) -> bool:
             state.prog_items[self.player]["RUPEES"] -= self.rupees[item.name]
         return change
 
+    # Same fill choices and weights used in LADXR.itempool.__randomizeRupees
+    filler_choices = ('Bomb', 'Single Arrow', '10 Arrows', 'Magic Powder', 'Medicine')

```

Comment:
This file almost entirely uses double-quotes
```suggestion
    filler_choices = ("Bomb", "Single Arrow", "10 Arrows", "Magic Powder", "Medicine")
```